src/node_internals.h

```diff
@@ -357,6 +357,19 @@ v8::MaybeLocal<v8::Object> New(Environment* env,
 // Mixing operator new and free() is undefined behavior so don't do that.
 v8::MaybeLocal<v8::Object> New(Environment* env, char* data, size_t length);
 
+inline
+v8::MaybeLocal<v8::Uint8Array> New(Environment* env,
```

Comment:
Would it be possible to explicitly note that a HandleScope must have been opened before calling this function?

Reply:
@TimothyGu I think that’s kind of explicit in the fact that this returns a `MaybeLocal`; that’s never valid without a `HandleScope`, right?

(Or, the other way around: The same is true of the other buffer constructors, but we don’t mention it there.)

Reply:
That's certainly true. But the other Buffer constructors open a HandleScope themselves so I was wondering.

Reply:
@TimothyGu I removed it in this case because, as far as I can tell, all the temporary handles created here refer to objects that are kept alive be the returned `Buffer` instance anyway.

If my understanding is correct, I think they could be removed in the other cases as well, but since those, but I am not 100 % sure and since some of them are public API I left them the way they were.